content/cn/template.json

```diff
@@ -9,5 +9,21 @@
   "org-link":"GitHub Org",
   "irc-link":"IRC Chat",
   "irc-logs-link":"Logs",
-  "gov-link":"项目管理"
+  "gov-link":"项目管理",
+  "verbose_version": "{{project.current_version}} 版本",
+  "downloads": {
+    "all": "其他"
+  },
+  "home": {
+    "download_links": "下载 {{> current_download_links}} 版本。",
+    "nightly_releases": "{{link '每日构建版本' 'https://iojs.org/download/nightly/'}} 可用于测试。",
+    "short_description": "{{link 'website'}} 是一个衍生自 {{link 'nodejs'}}，并兼容 {{link 'npm'}} 的开发平台。",
+    "slogan": "将 {{link 'pages.es6'}} 带入 Node 社区！"
+  },
+  "links": {
+    "pages": {
```

Comment:
Good point. I'd eventually like the per-release Changelog entries brought in as content on the site -- perhaps something like /cn/versions/v1.5.0/changelog.html . This way the i18n groups would have a place to publish the content as an alternative to the English changelog.

It wouldn't be too difficult to bring in the existing content (English and translated) as the notes are already formatted in Markdown.